build.gradle

```diff
@@ -2,7 +2,7 @@ plugins {
     id 'java'
     id 'maven-publish'
     id 'jacoco'
-    id "io.freefair.lombok" version "5.3.0"
+    //id "io.freefair.lombok" version "5.3.0"
```

Comment:
WHY?? WE NEED THIS??

Reply:
That was temporary. I need to figure out how to make a task that makes this togglable, as if you have it commented out then you can jump right to syntax errors, otherwise you cannot.